### PR TITLE
Add last_daily_round to PatientConsultation

### PR DESCRIPTION
**Pull request by shaileshaanand**
This PR adds last_daily_round to PatientConsultation as it is required to get bed details from patient consultation

I could not test the custom migration `0283_populate_patientconsultation_last_daily_round.py`, please test it before merge.

### CODE REVIEW

**Comment by vigneshhari** [comment]
Create a custom migration/ management command to update all the existing records.

**Comment by shaileshaanand** [comment]
> Create a custom migration/ management command to update all the existing records.

@vigneshhari Added

**Comment by shaileshaanand** [comment]
@vigneshhari I have updated the migration 0283 to use `bulk_update()`

**Comment by vigneshhari** [comment]
@shaileshaanand Django's migrations has capability to move up or down the migration versions, use that to test the migration out.

File: care/facility/migrations/0283_populate_patientconsultation_last_daily_round.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+def populate_last_daily_round(apps, *args):
+    PatientConsultation = apps.get_model("facility", "PatientConsultation")
+    DailyRound = apps.get_model("facility", "DailyRound")
+    consultations = PatientConsultation.objects.all()
+    for consultation in consultations:
+        consultation.last_daily_round = DailyRound.objects.filter(
+            patient_consultation_id=consultation.id
+        ).order_by('-date').first()
```
Comment (vigneshhari) [inline care/facility/migrations/0283_populate_patientconsultation_last_daily_round.py:11]: Use **created_date** here

**Review by vigneshhari — CHANGES_REQUESTED** [review]

**Comment by vigneshhari** [comment]
Make sure you test it out as well, once that's done we can merge in the change.

**Comment by shaileshaanand** [comment]
> Make sure you test it out as well, once that's done we can merge in the change.

@vigneshhari I have tested it and it works, the modified date is not mutated as well.

**Pull request merged**